documentation/adding-new-rtypes-rdata.md

```diff
@@ -0,0 +1,538 @@
+# Adding DNS Resource Types the "Rdata" way
```

Comment:
**Change**: Don't forget to add this page to the table or content:

https://github.com/StackExchange/dnscontrol/blob/e6ed65c70f57d79c6d578624934b9fbda5225b67/documentation/SUMMARY.md#L181-L186

Reply:
Fixed